File: GregorianToJalaliTest.dart
import 'GregorianToJalali.dart';

void main() {
  print(GregorianToJalali.gregorianToJalaliStr(2022, 1, 7) == "1400/10/17");
  print(GregorianToJalali.gregorianToJalaliStr(2022, 3, 20) == "1400/12/29");
}

File: GregorianToJalaliTest.py
import GregorianToJalali as GregorianToJalali
from datetime import datetime

# getting current date
year = datetime.now().year
month = datetime.now().month 
day = datetime.now().day  


g = GregorianToJalali.GregorianToJalali()

print(g.gregorianToJalaliStr(2022,1,7))

print(g.gregorianToJalaliStr(year,month,day))

